augur/cli/db.py

```diff
@@ -81,6 +82,47 @@ def add_repo_groups(augur_app, filename):
             else:
                 logger.info(f"Repo group with ID {row[1]} for repo group {row[1]} already exists, skipping...")
 
+@cli.command('add-github-org')
+@click.argument('organization_name')
+@pass_application
+def add_github_org(augur_app, organization_name):
+    """
+    Create new repo groups in Augur's database
+    """
+    org_query_response = requests.get(f"https://api.github.com/orgs/{organization_name}").json()
+    if "login" in org_query_response:
+        logger.info(f"Organization \"{organization_name}\" found")
+    else:
+        logger.fatal(f"No organization with name {organization_name} could be found")
+        exit(1)
+
+    page = 1
+    headers = {'Authorization': 'token %s' % augur_app.config.get_value("Database", "key")}
+    all_repos = []
+    repo_query_response = requests.get(org_query_response['repos_url'] + f"?per_page=100&page={page}", headers=headers).json()
+    while repo_query_response != []:
+        for repo in repo_query_response:
+            all_repos.append(repo)
+
+        page+=1
+        repo_query_response = requests.get(org_query_response['repos_url'] + f"?per_page=100&page={page}", headers=headers).json()
```

Comment:
page incrementation should be after you hit the endpoint

Reply:
It increments it after because it hits it once first and then needs to get the second request inside the loop, but looking back at it now that was a really wacky way to implement the loop and I'm gonna refactor it.